internal/provider/types/datasource/postgres.go

```diff
@@ -0,0 +1,16 @@
+package datasource
+
+import (
+	"terraform-provider-render/internal/provider/postgres"
+
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+)
+
+var DiskSizeGB schema.Int64Attribute = schema.Int64Attribute{
+	Description:         "Disk size in GB.",
+	MarkdownDescription: "Disk size in GB.",
```

Comment:
This field is only needed if it includes markdown that's not included in `Description`.